src/components/page-environment/active-component-status.js

```diff
@@ -22,29 +22,33 @@ export const ActiveComponentStatus = ({
     )
   ) {
     return (
-      <Chip type="danger">
-        <FontAwesomeIcon icon={faExclamationCircle} /> Missing secrets
-      </Chip>
+      <JobStatusChip type="failed">
```

Comment:
use 'type={ jobStatus.FAILED }' instead of adding your own icon here as this will give you the same effect